### DISCUSSION TITLE
Issuing DOIs for datasets hosted on Planetary Computer

### DISCUSSION THREAD

**Discussion by maxrjones** [microsoft/PlanetaryComputer#124] [Ideas]
Thanks for your work on Planetary Computer!

Would you consider issuing DOIs for the datasets including in the Planetary Computer Data Catalog? This would help people cite specific datasets.

For context, [CarbonPlan](https://carbonplan.org/) is exploring options for associating a DOI with our [downscaled climate data](https://carbonplan.org/research/cmip6-downscaling), which we anticipate will be included in the data catalog shortly (xref. https://github.com/microsoft/PlanetaryComputerExamples/pull/223). The most common mechanisms for acquiring a dataset DOI, such as [Zenodo](https://zenodo.org/) or [Dryad](https://datadryad.org/), are not ideal due to size limitations for those repositories.

The [recent partnership](https://huggingface.co/blog/introducing-doi) between [Hugging Face](https://huggingface.co/) and [DataCite](https://datacite.org/) provides some insight into the steps required for a data platform to directly issue DOIs. https://github.com/huggingface/hub-docs/issues/25 provides background on that partnership.

**Comment by TomAugspurger** [comment]
Hi Max,

We do have a DOI you can use at https://zenodo.org/record/7261897 for the Planetary Computer generally. We don't have anything for individual datasets, and it'll be some time before I can look into that. For now, I wonder if some combination of our DOI and perhaps a reference to the dataset producer will together give you something you can use.
